akka/src/main/resources/www/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html ng-app="webClient">
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
it could, but the index.html is not going through the code, we just serve it as resource..
so the import script line would be like
 <script src="/webjars/ajax/libs/jquery/1.11.3/jquery.min.js"></script>

so the version number would be duplicated.